datadog/resource_datadog_security_monitoring_default_rule.go

```diff
@@ -197,6 +231,24 @@ func buildSecMonDefaultRuleUpdatePayload(currentState datadogV2.SecurityMonitori
 		payload.Cases = &updatedRuleCase
 	}
 
+	tfFilters := d.Get("filter").([]interface{})
+	payloadFilters := make([]datadogV2.SecurityMonitoringFilter, len(tfFilters))
+
+	for idx, tfRuleFilter := range tfFilters {
+		structRuleFilter := datadogV2.SecurityMonitoringFilter{}
+
+		ruleFilter := tfRuleFilter.(map[string]interface{})
+		action := datadogV2.SecurityMonitoringFilterAction(ruleFilter["action"].(string))
+		structRuleFilter.Action = &action
```

Comment:
We may want to use the setter for the action here, too. Filters don't necessarily always have an action set. 

```suggestion
		if action, ok := ruleFilter["action"]; ok {
			structRuleFilter.SetAction(ruleFilter["action"].(datadogV2.SecurityMonitoringFilterAction))
		}
```